### PR TITLE
add user saved drink route,

### PR DESCRIPTION
**Pull request by JakeKeels**
This requires review and further testing before being fully committed to the main branch.

### CODE REVIEW

File: routes/create/createSavedDrink.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const createSavedDrink = async (request, db_connection) => {
+
+    const id = parseInt(request.payload.user_id);
+    const drink = parseInt(request.payload.drink);
+    return await new Promise((resolve, reject) => {
+
+        const sql_query = `
+      insert into saved_drinks(user_id,drink_id) values('${id}','${drink}')
+    `;
+        console.log(update);
```
Comment (anthonymesa) [inline routes/create/createSavedDrink.js:18]: is 'update' an artifact from other code?

File: routes/create/createSavedDrink.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        return new Promise((resolve, reject) => {
+            const request = new Request(update,
```
Comment (anthonymesa) [inline routes/create/createSavedDrink.js:20]: I see now, you should replace 'update' with sql_query.

File: routes/create/createSavedDrink.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                (err, rowCount) => {
+                    if (err) {
+                        console.log(err);
+                        resolve(createResponse(-1, null));
+                    } else {
+                        console.log(rowCount);
+                        resolve(createResponse(0, rowCount == 1));
```
Comment (anthonymesa) [inline routes/create/createSavedDrink.js:27]: Will the updating of a table actually return any rows? If not, we can be fine to resolve null here given the status value is 0

File: routes/update/updateReadyStatus.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  return new Promise((resolve, reject) => {
+    //  Create dabase request to count from test table (should be 1)
+    const request = new Request(update,
+      (err, rowCount) => {
+        if (err) {
+          console.log(err);
+          resolve(createResponse(-1, null));
+        } else {
+          console.log(rowCount);
+          resolve(createResponse(0, rowCount == 1));
```
Comment (anthonymesa) [inline routes/update/updateReadyStatus.js:27]: Again, if no rows are returned, we can just resolve with a value of null in the createResponse function

File: server.js [hunk 5]
```diff
@@ -24,6 +24,7 @@ const { getUser } = require("./routes/get/getUser");
 const { getTabDrinks } = require("./routes/get/getTab");
 const { getSavedDrinks } = require("./routes/get/getSavedDrinks");
 const { getBarDrinks } = require("./routes/get/getBarDrinks");
+const { createSavedDrink } = require("./routes/create/createSavedDrink");
```
Comment (anthonymesa) [inline server.js:28]: updateReadyStatus needs to be set to a route

**Review by anthonymesa — CHANGES_REQUESTED** [review]
really only minor changes!

**Comment by JakeKeels** [comment]
Fixed the minor issues, should be ok to pull

**Pull request merged**